apps/api/src/app/events/usecases/send-message/send-message-type.usecase.ts

```diff
@@ -13,12 +14,22 @@ export abstract class SendMessageType {
     message,
     status: 'error' | 'sent' | 'warning',
     errorId: string,
-    errorMessage: string,
+    errorMessageFallback: string | any,
     command: SendMessageCommand,
     notification: NotificationEntity,
-    logCodeEnum: LogCodeEnum
+    logCodeEnum: LogCodeEnum,
+    error?: any
   ) {
-    await this.messageRepository.updateMessageStatus(message._id, status, null, errorId, errorMessage);
+    const errorString = (toStringify(error?.response?.body) ||
+      toStringify(error?.response) ||
+      toStringify(error) ||
+      JSON.stringify(errorMessageFallback)) as string;
```

Comment:
With my other suggestion I think this wouldn't be needed anymore and we can just pass the unmodified `errorMessageFallback`.